.travis.yml

```diff
@@ -9,8 +10,13 @@ before_script:
   - Rscript -e "install.packages('knitr', repos='http://cran.cnr.Berkeley.edu/')"
   - Rscript -e "install.packages('rmarkdown', repos='http://cran.cnr.Berkeley.edu/')"
 
+before_script:
+  - Rscript -e "install.packages('knitr')"		 +  - Rscript -e "install.packages('knitr',repos='http://cran.cnr.Berkeley.edu/')"
+  - Rscript -e "install.packages('rmarkdown')"		 +  - Rscript -e "install.packages('rmarkdown',repos='http://cran.cnr.Berkeley.edu/')"
+  
```

Comment:
this doesn't look right - the previous 3 lines are also `before_script`, and these lines look like GitHub conflict information not formatted for travis. delete these 3?

I agree that specifying cnr.Berkeley.edu as the repository is probably unnecessary - changing the previous 3 lines to just be `install.packages('knitr')`, etc. is good with me.